inc/order_item.class.php

```diff
@@ -1896,5 +1903,13 @@ public static function uninstallOrderItemNotification() {
       }
    }
 
+   public function getManufacturerReference($reference_id){
+     global $DB;
+     $query = "SELECT manufacturers_reference
```

Comment:
It would be better to use querying syntax as described [here](https://glpi-developer-documentation.readthedocs.io/en/master/devapi/database/dbiterator.html) instead of using raw SQL.